paper/paper.bib

```diff
@@ -126,3 +126,24 @@ @Manual{base2021
     url = {https://www.R-project.org/},
   }
 
+@article{nuijten2016prevalence,
+  title={The prevalence of statistical reporting errors in psychology (1985--2013)},
+  author={Nuijten, Mich{\`e}le B and Hartgerink, Chris HJ and Van Assen, Marcel ALM and Epskamp, Sacha and Wicherts, Jelte M},
+  journal={Behavior research methods},
+  volume={48},
+  pages={1205--1226},
+  year={2016},
+  publisher={Springer},
+  doi={doi.org/10.3758/s13428-015-0664-2}
+}
+
+@software{easystatspackage,
```

Comment:
It seems there are currently two references for easystats, which one is correct?

``` r
citation("easystats")
#>   Lüdecke, Patil, Ben-Shachar, Wiernik, Bacher, Thériault, & Makowski
#>   (2022). easystats: Framework for Easy Statistical Modeling,
#>   Visualization, and Reporting. CRAN. Available from
#>   https://easystats.github.io/easystats/
report::cite_easystats()
#> - Lüdecke, D., Makowski, D., Ben-Shachar, M. S., Patil, I., Wiernik, B. M.,
#>     Bacher, Etienne, & Thériault, R. (2023). easystats: Streamline model
#>     interpretation, visualization, and reporting (0.6.0.7) [R package].
#>     https://easystats.github.io/easystats/ (Original work published 2019)
```

<sup>Created on 2023-02-26 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
Second one. 

We should update the first citation.